src/app/[locale]/viewer/[base64AasId]/page.tsx

```diff
@@ -44,18 +44,18 @@ export default function Page() {
 
     async function loadAasContent() {
         if (repoUrl) {
-            const repoAas = await getAasFromRepository(aasIdDecoded, repoUrl);
-            setAas(repoAas ?? undefined);
+            const response  = await getAasFromRepository(aasIdDecoded, repoUrl);
+            setAas(response.result ?? undefined);
             return;
         }
 
         const result = await performFullAasSearch(aasIdDecoded);
```

Comment:
I'd like if we renamed "result" to "response" here for clearity to avoid the result.result call

Reply:
Done